# camera intrins
cam_topic: "/camera_array/cam0/image_raw"
lidar_topic: "/velodyne_points"

cam_intrins:
   [1709.163406, 0.000000, 622.365608, 
   0.000000, 1710.142290, 510.444884, 
   0.000000, 0.000000, 1.000000]

lidar2cam:
   [0.0443473,  -0.998957,   0.0108723, -0.16487,
    -0.0566353, -0.0133795, -0.998305,  -0.00636259, 
    0.99741,     0.0436564, -0.0571695, -0.11176, 
    0.0,         0.0,        0.0,        1]

